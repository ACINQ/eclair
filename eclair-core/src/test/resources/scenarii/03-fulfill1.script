# Simple test that we can fulfill (remove) an HTLC after fully settled.
# Initial state: A=1000000 sat, B=1000000 sat, both fee rates=10000 sat
A:offer 1000000,b8928207364d445daa42f4ba8be0ef661b8d7190206c01f6ee8281566b3dec0a
B:recvoffer
A:commit
B:recvcommit
A:recvrevoke
B:commit
A:recvcommit
B:recvrevoke

B:fulfill 0,60303ae22b998861bce3b28f33eec1be758a213c86c93c076dbe9f558c11c752
B:commit
A:recvremove
A:recvcommit
B:recvrevoke
A:commit
B:recvcommit
A:recvrevoke

checksync
echo ***A***
A:dump
echo ***B***
B:dump
