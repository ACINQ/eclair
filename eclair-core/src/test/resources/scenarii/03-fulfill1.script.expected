***A***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 999000000 msat
  Balance them: 1001000000 msat
  Fee rate: 10000
REMOTE COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 1001000000 msat
  Balance them: 999000000 msat
  Fee rate: 10000
***B***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 1001000000 msat
  Balance them: 999000000 msat
  Fee rate: 10000
REMOTE COMMITS:
 Commit 2:
  Offered htlcs:
  Received htlcs:
  Balance us: 999000000 msat
  Balance them: 1001000000 msat
  Fee rate: 10000
